consensus/config.go

```diff
@@ -38,7 +38,7 @@ func NewConfig(dataDir string, rpcListenPort uint, p2pListenPort uint, proxyList
 	tendermintCfg.SetRoot(dataDir)
 	tendermintCfg.RPC.ListenAddress = fmt.Sprintf("tcp://0.0.0.0:%d", rpcListenPort)
 	tendermintCfg.P2P.ListenAddress = fmt.Sprintf("tcp://0.0.0.0:%d", p2pListenPort)
-	tendermintCfg.ProxyApp = fmt.Sprintf("tcp://127.0.0.1:%d", proxyListenPort)
+	tendermintCfg.ProxyApp = "lightchain"
```

Comment:
The `proxyApp` value("lightchain") should an argument of the `NewConfig()` to be consistent with the initialization of the config object. 

Also keep in mind after your change `proxyListenPort` and `proxyProtocol` is useless, so you should also clean them and most likely replace it by `proxyApp`

Reply:
That's a good point actually. Didn't notice the var was unused. Changed.